syncer.go

```diff
@@ -57,6 +57,8 @@ func (s *syncer) start(wg *sync.WaitGroup) {
 	go func() {
 		defer wg.Done()
 		var requests []*syncRequest
+		timer := time.NewTimer(s.interval)
```

Comment:
In the syncer, would it be more appropriate for the initialized Interval to be 0 while the Reset to then use the s.interval value?